content/docs/digging_deeper/transmit.md

```diff
@@ -0,0 +1,373 @@
+# Transmit
+
+Transmit is a native Server-Sent-Event (SSE) module built for AdonisJS. It is a simple and efficient way to send real-time updates to the client, such as notifications, live chat messages, or any other type of real-time data.
```

Comment:
And very opinionated about how dev should use SSE. (endpoint is fixed and you have to create a listener and then subscribe to channel)